EarthAsDMAnalyzer/plugins/EarthAsDMAnalyzer.cc

```diff
@@ -204,6 +207,20 @@ class EarthAsDMAnalyzer : public edm::one::EDAnalyzer<edm::one::SharedResources>
   std::vector<float> muon_dtSeg_eta_;
   std::vector<float> muon_dtSeg_phi_;
 
+  std::vector<float> Track_Reconsruction_Score_;
+  std::vector<float> muon_dtSeg_globY_Hits_rPhi_; //dtGlobalPointYValues
```

Comment:
```suggestion
  std::vector<float> muon_dtSeg_rPhi_globY_;
```
* these are not at the hit level